packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ColorUtils.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+private const val ALPHA_CODE_LENGTH: Int = 2
+private const val HEX_RADIX: Int = 16
+private const val OPAQUE_ALPHA_VALUE: Int = 255
+private const val HEX_COLOR_INCLUDING_ALPHA_LENGTH: Int = 8
+
+internal fun resolveAlphaValue(viewAlpha: Float, backgroundColor: Int): Int {
+    // must not be toHexString because we'll lose the sign bit
+    val bgColorAsHexCode = backgroundColor.toString(HEX_RADIX)
+    var pct = viewAlpha
```

Comment:
I don't know what `pct` is, could you make the variable name more explicit or add a comment if it is a concept in color?

Reply:
I'll change it to opacityPercentage - a value between 0 and 1